src/lib/components/messages/inbox/conversation_snippet.tsx

```diff
@@ -122,3 +125,31 @@ const styles = StyleSheet.create<Styles>({
     borderRadius: 4,
   },
 })
+
+export default Relay.createContainer(ConversationSnippet, {
+  fragments: {
+    conversation: () => Relay.QL`
+      fragment on ConversationType {
+        id
+        inquiry_id
+        from_name
+        from_email
+        to_name
+        last_message
+        artworks {
```

Comment:
It looks like this query is fetching potentially multiple artworks, but [only 1 gets used](https://github.com/artsy/emission/blob/2a2167a083f430867c73bfd834b041ab12c50fc7/src/lib/components/messages/inbox/conversation_snippet.tsx#L79). Is that correct?